tests/subsys/dsp/common/math_helper.h

```diff
@@ -0,0 +1,276 @@
+/* ----------------------------------------------------------------------
```

Comment:
Would it make sense to only keep a copy of these and `test_common.h`?

Reply:
Done, I kept just one copy and used target_include_directories in the new test location.